apps/src/templates/projects/StartNewProject.jsx

```diff
@@ -0,0 +1,108 @@
+import React, {PropTypes} from 'react';
+import NewProjectButtons from './NewProjectButtons.jsx';
+import i18n from '@cdo/locale';
+import Button from '../Button';
+import color from '../../util/color';
+import {valueOr} from '../../utils';
+
+const styles = {
+  button: {
+    float: 'right',
+    marginRight: 1,
+  },
+  headingStartNew: {
+    paddingRight: 10,
+    paddingBottom: 10,
+    fontSize: 16,
+    fontFamily: '"Gotham 4r"',
+    color: color.charcoal,
+    marginBottom: -10
+  },
+  spacer: {
+    paddingTop: 10,
+    clear: 'both',
+    width: '100%'
+  }
+};
+
+class StartNewProject extends React.Component {
+  static propTypes = {
+    projectTypes: PropTypes.arrayOf(PropTypes.string),
+    isRtl: PropTypes.bool,
+    canViewFullList: PropTypes.bool,
+    canViewAdvancedTools: PropTypes.bool, // Default: true
+  };
+
+  state = {
+    showFullList: false,
+  };
+
+  toggleShowFullList = () => {
+    this.setState({showFullList: !this.state.showFullList});
+  };
+
+  render() {
+    const { canViewFullList, isRtl } = this.props;
+    const canViewAdvancedTools = valueOr(this.props.canViewAdvancedTools, true);
```

Comment:
This is a little bit odd.  I feel like the React-y way to do this would be with `defaultProps`:
```jsx
static defaultProps = {
  canViewAdvancedTools: true
};
```

Reply:
Ahh, I was not aware of defaultProps. This is much better!